app/src/layouts/common/sidebar/organizationsControl/organizationsPopover/organizationsPopover.jsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import { useIntl, defineMessages } from 'react-intl';
+import classNames from 'classnames/bind';
+import { useSelector } from 'react-redux';
+import { availableProjectsSelector } from 'controllers/user';
+import { urlOrganizationAndProjectSelector } from 'controllers/pages';
+import { ScrollWrapper } from 'components/main/scrollWrapper';
+import Link from 'redux-first-router-link';
+import { PROJECT_SETTINGS_PAGE } from 'controllers/pages/constants';
+import { OrganizationsItem } from './organizationsItem';
+import styles from './organizationsPopover.scss';
+
+const cx = classNames.bind(styles);
+
+const messages = defineMessages({
+  allOrganizations: {
+    id: 'OrganizationsPopover.allOrganizations',
+    defaultMessage: 'All organizations',
+  },
+});
+
+export const OrganizationsPopover = () => {
+  const { formatMessage } = useIntl();
+  const availableProjects = useSelector(availableProjectsSelector);
+  const { organizationSlug: currentOrganization, projectSlug } = useSelector(
+    urlOrganizationAndProjectSelector,
+  );
+
+  return (
+    <div className={cx('organizations-popover')}>
+      {availableProjects.length > 0 && (
+        <div className={cx('all-organizations')}>
+          <Link
+            to={{
+              type: PROJECT_SETTINGS_PAGE,
+              payload: { organizationSlug: currentOrganization, projectSlug },
+            }}
+            className={cx('all-organizations-link')}
+          >
+            {formatMessage(messages.allOrganizations)}
+          </Link>
+        </div>
+      )}
+      <ScrollWrapper
+        autoHide
+        autoHeight
+        autoHeightMax={window.innerHeight - 195}
```

Comment:
What is `195` here? Try to avoid magic numbers as they are hard to understand and maintain.
Does this value agreed with design team?

Reply:
Done.